lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -152,7 +152,9 @@ def scrub(transcript)
       }
 
       def add_extra_data(post, payment, options)
-        post[:telephoneNumber] = options[:billing_address][:phone] if options[:billing_address][:phone]
```

Comment:
Would this simplify by using:
post[:telephoneNumber] = options[:billing_address][:phone] if options.dig( :billing_address, :phone)

Reply:
Yep! Updated.